google/ggrc-core
# GGRC-4570 Export Attribute does not contain ticket tracking number

# Dependencies

This PR is `on hold` until the dependencies are merged:
- [x] #8066

# Issue description

During export of an issue the link to issue tracker should be exported as well  

# Steps to test the changes

1. Create an issue
2. Add issue tracker info to the issue
3. Export the issue
Expected result: the link to Ticket Tracker is present in exported csv

# Solution description

Add a column to exported aliases for Issue

# Sanity checklist


# PR Review checklist
